workspace/__lib__/xod/debug/receive-byte/patch.cpp

```diff
@@ -0,0 +1,22 @@
+#pragma XOD evaluate_on_pin disable
+#pragma XOD evaluate_on_pin enable input_READ
+
+struct State {
+};
+
+// clang-format off
+{{ GENERATED_CODE }}
+// clang-format on
+
+void evaluate(Context ctx) {
+    if (!isInputDirty<input_READ>(ctx))
+        return;
+
+    auto inet = getValue<input_INET>(ctx);
+    char r = 'Z';
```

Comment:
```suggestion
    char r;
```

Why spend extra resources to set something unused?